python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -150,6 +150,9 @@ def _get_ray_cr_no_cpu_error() -> dict:
     del cr["spec"]["workerGroupSpecs"][0]["template"]["spec"]["containers"][0][
         "resources"
     ]["limits"]["cpu"]
+    del cr["spec"]["workerGroupSpecs"][0]["template"]["spec"]["containers"][0][
+        "resources"
+    ]["requests"]["cpu"]
```

Comment:
We now need to delete `requests` as well to satisfy the `_get_ray_cr_no_cpu_error` test.